state-chain/chains/src/lib.rs

```diff
@@ -496,6 +496,7 @@ pub trait ExecutexSwapAndCall<C: Chain>: ApiCall<C::ChainCrypto> {
 		source_address: Option<ForeignChainAddress>,
 		gas_budget: C::ChainAmount,
 		message: Vec<u8>,
+		cf_parameter: Vec<u8>,
```

Comment:
Rename to `cf_parameters` to match the State Chain code. Same for other instances of this.